### PR TITLE
CFE-2283: Document email filtering feature in cf-execd.

### PR DESCRIPTION
**Pull request by kacf**

### CODE REVIEW

**Comment by kacf** [comment]
@estenberg: Does this match expectations?

**Comment by estenberg** [comment]
Yes, very nice!

File: reference/components/cf-execd.markdown [hunk 1]
```diff
@@ -123,6 +123,66 @@ symbols may be used if desired.
 
     exec_command => "$(sys.workdir)/bin/cf-agent -f failsafe.cf && $(sys.workdir)/bin/cf-agent";
 
+### mailfilter_exclude
+
+**Description:** List of anchored regular expressions that, if matched by a log
```
Comment (nickanderson) [inline reference/components/cf-execd.markdown]: We can make anchored a link
``` 
[anchored][anchored]
```

File: reference/components/cf-execd.markdown [hunk 2]
```diff
@@ -123,6 +123,66 @@ symbols may be used if desired.
...
+        # Ignore agent execution emails about permission errors.
+        mailfilter_exclude => { ".*Permission denied.*" };
+    }
+```
+
+**History:** Introduced in CFEngine 3.9.
+
+### mailfilter_include
+
+**Description:** List of anchored regular expressions that must match a log
```
Comment (nickanderson) [inline reference/components/cf-execd.markdown]: We can make anchored a link

```
[anchored][anchored]
```

**Comment by nickanderson** [comment]
Nice. One minor suggestion for linking anchored to a page describing it.

File: reference/components/cf-execd.markdown (hunk 1 above)
Comment (kacf) [inline reference/components/cf-execd.markdown]: Thanks, fixed in both places!

**Pull request merged**